coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -179,23 +179,25 @@ def _compile_multi_match_regex(strings):
     :param strings: The strings to match.
     :return:        A regex object.
     """
-    return re.compile('|'.join(re.escape(s) for s in strings))
+    return re.compile('^\s*(?P<marker>' +
+                      ('|'.join(re.escape(s) for s in strings)) +
+                      ')')
```

Comment:
I like the parent name `_compile_multi_match_regex` as we are adding escape sequences over each character of the docstring. Let me write a description inside the docstring :+1: